### PR TITLE
Delete repeating log without value

### PR DESCRIPTION
**Pull request by Barakmor1**
**What this PR does / why we need it**:
Currently MTQ Operator is using GenerateContainers func to evaluate the target pod and figure out if it can release blocked migration, the current log result repeating logging which make it hard to debug.
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by iholder101** [comment]
Looks reasonable to me.
/lgtm

**Review by enp0s3 — COMMENTED** [review]
/approve

File: pkg/container-disk/container-disk.go [hunk 1]
```diff
@@ -335,7 +335,7 @@ func generateContainerFromVolume(vmi *v1.VirtualMachineInstance, config *virtcon
 		copyPathArg := path.Join(volumeMountDir, mountedDiskName)
 		args = []string{"--copy-path", copyPathArg}
 
-		log.Log.Object(vmi).Infof("arguments for container-disk \"%s\": --copy-path %s", name, copyPathArg)
+		log.Log.V(3).Object(vmi).Infof("arguments for container-disk \"%s\": --copy-path %s", name, copyPathArg)
```
Comment (xpivarc) [inline pkg/container-disk/container-disk.go:338]: @Barakmor1 Let's remove this, I don't see value in this log. You will have the arguments in the Pod spec.

**Review by xpivarc — CHANGES_REQUESTED** [review]

File: pkg/container-disk/container-disk.go (hunk 1 above)
Comment (enp0s3) [reply pkg/container-disk/container-disk.go:338]: +1

Comment (Barakmor1) [reply pkg/container-disk/container-disk.go:338]: Yep i think you are right

**Review by enp0s3 — APPROVED** [review]

**Comment by xpivarc** [comment]
/hold
you need to run make generate

**Comment by Barakmor1** [comment]
/unhold
@xpivarc no reason to hold if required test are failing

**Comment by brianmcarey** [comment]
> /unhold @xpivarc no reason to hold if required tests are failing

A hold can be handy to stop the retest bot from needlessly retesting a PR that won't pass

**Comment by xpivarc** [comment]
> /unhold @xpivarc no reason to hold if required tests are failing

Actually, this PR is approved&lgtmed so there would be an automatic retest without it

**Pull request merged**